go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/husniadil/uuid
 go 1.15
 
 require (
-	github.com/google/uuid v1.2.0
+	github.com/google/uuid v1.3.0
```

Comment:
The `NullUUID` feature added in this version can be beneficial for handling null values in JSON and SQL. Ensure that your codebase is updated to utilize this feature where appropriate, as it can simplify handling of optional UUIDs. However, be cautious about how you handle null values to avoid potential null pointer dereferences.